demo/src/index.js

```diff
@@ -1,18 +1,12 @@
 import React, { Component } from 'react';
 import * as d3 from 'd3';
-import Promise from 'bluebird';
 import { render } from 'react-dom';
-import {
-  DataProvider,
-  ChartContainer,
```

Comment:
It seems there is no need anymore for the ChartContainer component from a separate file

Reply:
Yep, ChartContainer will be deleted once. Cleanup is left.